src/r502_commands.c

```diff
@@ -140,6 +140,15 @@ int32_t populate_command_args(Command *command, int32_t arg_num, va_list ap) {
         command->body.store.buf = va_arg(ap, uint32_t);
         command->body.store.index = va_arg(ap, uint32_t);
 
+        break;
+    case AuraLedConfig:
```

Comment:
Keep the same order of the commands as in the `cmd_func` array

Reply:
Done!